docker/policies/apt_get_missing_yes_flag_to_avoid_manual_input.rego

```diff
@@ -0,0 +1,74 @@
+package appshield.dockerfile.DS021
+
+import data.lib.docker
+
+__rego_metadata__ := {
+	"id": "DS021",
+	"title": "'apt-get' is missing '-y' to avoid manual input",
+	"version": "v1.0.0",
+	"severity": "HIGH",
+	"type": "Dockerfile Security Check",
+	"description": "Check if apt-get calls use the flag -y to avoid user manual input.",
+	"recommended_actions": "Add -y flag to apt-get",
+	"url": "https://docs.docker.com/engine/reference/builder/#run",
+}
+
+__rego_input__ := {
+	"combine": false,
+	"selector": [{"type": "dockerfile"}],
+}
+
+short_flags := `(-([a-xzA-XZ])*y([a-xzA-XZ])*)`
+
+long_flags := `(--yes)|(--assume-yes)`
+
+combined_flags := sprintf(`(%s|%s)`, [short_flags, long_flags])
+
+deny[res] {
+	args := get_apt_get[_]
+	res := sprintf("-y flag is missed: %s", [args])
+}
+
+get_apt_get[arg] {
+	run = docker.run[_]
+
+	count(run.Value) == 1
+	arg := run.Value[0]
+
+	is_apt_get(arg)
+
+	not includes_assume_yes(arg)
+}
+
+#checking json array
+get_apt_get[arg] {
+	run = docker.run[_]
+
+	count(run.Value) > 1
+
+	arg := concat(" ", run.Value)
+
+	is_apt_get(arg)
+
+	not flag_includes_assume_yes(run.Value)
```

Comment:
We already concat `run.Value`, so what if using the same function `includes_assume_yes`? It reduces a possibility of a bug.